src/styles/application/utilities/_patternfly4.scss

```diff
@@ -6,12 +6,49 @@
 // This doesn't affect PF3.
 // https://github.com/twbs/bootstrap/blob/v3.4.0-dev/less/scaffolding.less#L23
 
+// RFS Imports
+@import "../../../../node_modules/rfs/scss.scss";
+
 /* stylelint-disable */
 /* allows the override of default styles */
 html {
   font-size: unset !important; // the important is needed because we don't know if pf3 will be loaded after pfnext
 }
 
+// $pf-global--FontSize--4xl: 2.25rem !default;
```

Comment:
Was leaving these commented lines in intentional? Might want to add a quick comment a line above to say why, otherwise delete.

Reply:
Those were left in on accident - I've removed the commented out code.